docs/testnet/deploying-a-smart-contract.md

```diff
@@ -3,12 +3,12 @@ Using the steps below you will add an extension to your MetaMask wallet so it ca
 
 ## Prerequisites
 * [MetaMask](https://metamask.io/) wallet installed in your browser.
-* A local copy of the Obscuro MetaMask wallet extension downloaded from the [releases area](https://github.com/obscuronet/go-obscuro/releases) of the Obscuro repository on GitHub.
+* A local copy of the [Obscuro MetaMask wallet extension](wallet-extension.md)
```

Comment:
I thought it was better not to repeat the instructions about where to get the wallet extension, and just link to the wallet extension page. Otherwise, you'd also have to repeat the instructions for compiling your own binary in some cases, and it got a bit messy.